### PR TITLE
RNFS2 Examples

### PR DESCRIPTION
**Pull request by ian-wd**

### CODE REVIEW

File: example/App/example1.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  /**
+   * Methods
+   */
+  const getFolderText = () => {
+    if (Platform.OS === 'ios') {
+      return 'DocumentDirectory';
+    }
+
+    return 'DownloadDirectory';
+  };
```
Comment (iBotPeaches) [inline example/App/example1.tsx:20]: This method could probably move to that utils file you have.

File: example/App/example1.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  const executeExample = async () => {
+    try {
+      // cleanup previous result
+      setResult('');
+
+      setRunningAction(true);
+      if (Platform.OS === 'android') {
+        const granted = await PermissionsAndroid.request(PermissionsAndroid.PERMISSIONS.WRITE_EXTERNAL_STORAGE, {
+          title: 'Example Read/Write Storage Permission',
+          message: 'Examoke App needs read/write access to your phone storage to save files',
```
Comment (iBotPeaches) [inline example/App/example1.tsx:31]: `Example`

File: example/App/example2.tsx [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+  const executeExample = async () => {
+    try {
+      // cleanup previous result
+      setResult('');
+
+      setRunningAction(true);
+      if (Platform.OS === 'android') {
+        const granted = await PermissionsAndroid.request(PermissionsAndroid.PERMISSIONS.WRITE_EXTERNAL_STORAGE, {
+          title: 'Example Read/Write Storage Permission',
+          message: 'Examoke App needs read/write access to your phone storage to save files',
```
Comment (iBotPeaches) [inline example/App/example2.tsx:31]: `Example`

File: example/App/example2.tsx [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+  /**
+   * Methods
+   */
+  const getFolderText = () => {
+    if (Platform.OS === 'ios') {
+      return 'DocumentDirectory';
+    }
+
+    return 'DownloadDirectory';
+  };
```
Comment (iBotPeaches) [inline example/App/example2.tsx:20]: Same function we could migrate to utils.

File: example/package.json [hunk 5]
```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "react": "18.1.0",
     "react-native": "0.70.6",
-    "react-native-fs2": "link:../"
+    "react-native-fs2":  "file:../"
```
Comment (iBotPeaches) [inline example/package.json:15]: Looks like an extra space here now

**Review by iBotPeaches — APPROVED** [review]

**Review by old — APPROVED** [review]

**Pull request merged**